brave/brave-core
# Attempt 3: fall through to upstream logic that allows some videos to autoplay muted

Resolves brave/brave-browser#9597

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

### Testing block
1. Go to https://vimeo.com/355789315
2. Set autoplay to block
3. Reload the tab
4. Video should be paused
5. Click on the play button of the video and it should be able to play

### Testing brave together
1. Open http://together.brave.com/
2. Connect with other peer
3. Should be abled to see video on bottom right rectangle and hear from the peer

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
